src/components/UI/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import { MouseEventHandler } from 'react';
+import styles from './Button.module.css';
+
+interface ButtonProps {
+  filled?: boolean;
+  children: string;
+  onClick?: MouseEventHandler;
+}
+
+const Button: React.FC<ButtonProps> = (props) => {
+  return (
+    <button
+      onClick={props.onClick}
+      className={`${styles.button}  ${
+        props.filled && styles['button--filled']
```

Comment:
**nitpick:** You can use https://github.com/JedWatson/classnames, it can help you a lot with working with class names